Sources/XCTest/XCTestCase.swift

```diff
@@ -367,5 +367,43 @@ extension XCTestCase {
         
         return expectation
     }
+    
+    /// Creates and returns an expectation that is fulfilled if the predicate 
+    /// returns true when evaluated with the given object. The expectation
+    /// periodically evaluates the predicate and also may use notifications or 
+    /// other events to optimistically re-evaluate.
+    ///
+    /// - Parameter predicate: The predicate that will be used to evaluate the 
+    ///   object.
+    /// - Parameter object: The object that is evaluated against the conditions
+    ///   specified by the predicate.
+    /// - Parameter description: This string will be displayed in the test log
+    ///   to help diagnose failures.
+    /// - Parameter file: The file name to use in the error message if
+    ///   this expectation is not waited for. Default is the file
+    ///   containing the call to this method. It is rare to provide this
+    ///   parameter when calling this method.
+    /// - Parameter line: The line number to use in the error message if the
+    ///   this expectation is not waited for. Default is the line
+    ///   number of the call to this method in the calling file. It is rare to
+    ///   provide this parameter when calling this method.
+    /// - Parameter handler: A block to be invoked when evaluating the predicate
+    ///   against the object returns true. If the block is not provided the 
+    ///   first successful evaluation will fulfill the expectation. If provided,
+    ///   the handler can override that behavior which leaves the caller
+    ///   responsible for fulfilling the expectation.
+    public func expectation(forPredicate predicate: NSPredicate, evaluatedWithObject object: AnyObject, file: StaticString = #file, line: UInt = #line, handler: XCPredicateExpectationHandler? = nil) -> XCPredicateExpectation {
```

Comment:
Agreed. [This](https://github.com/modocache/XCTest.swift/blob/5b5a152150bacea9c373dcfb16a8384563cd4419/XCTest.swift#L85) was generated with the Swift 3 importer from a month ago, but it matches @briancroom's signature.